addons/studio-followers/addon.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Add followers to your studios",
+  "description": "Creates a modal that you can use to invite your followers to studios.",
+  "credits": [
+    {
+      "name": "9gr",
+      "link": "https://scratch.mit.edu/users/9gr"
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["https://scratch.mit.edu/studios_www/*"],
```

Comment:
This and the other url are incorrect - please replace both with "studios".

Reply:
Isn't this needed for the scratchr2 migration thing?

Reply:
Nope. In fact this wouldn't work.

Reply:
Oops, I forgot to update my fork before making a PR